pyproject.toml

```diff
@@ -45,7 +45,10 @@ include = ["tests/", "CHANGELOG.rst", "tox.ini"]
 src = ["src"]
 fix = true
 show-fixes = true
-show-source = true
+output-format = "full"
+
+[tool.ruff.format]
+docstring-code-format = true
```

Comment:
oh nice! does this mean we can remove blacken-docs from pre-commit? 🤔

Reply:
Not yet, I'm afraid. There seems to be a difference between
- [ ] [Support formatting of embedded code](https://github.com/astral-sh/ruff/issues/8237)
- [x] [format code examples in docstrings](https://github.com/astral-sh/ruff/issues/7146)

I picked this line from marshmallow's pyproject.toml without much thinking, TBH.

Reply:
ah woops i didn't realize this was already in marshmallow. looks like formatting RST docs isn't implemented yet, so we still need blacken-docs for now